static/app/views/explore/multiQueryMode/queryVisualizations/chart.tsx

```diff
@@ -295,9 +309,13 @@ export function MultiQueryModeChart({
       revealActions="always"
       Visualization={
         <TimeSeriesWidgetVisualization
-          dataCompletenessDelay={INGESTION_DELAY}
-          visualizationType={visualizationType}
-          timeSeries={chartInfo.data}
+          plottables={chartInfo.data.map(timeSeries => {
+            return new DataPlottableConstructor(timeSeries, {
+              delay: INGESTION_DELAY,
+              color: isTimeSeriesOther(timeSeries) ? theme.chartOther : undefined,
+              stack: 'all',
```

Comment:
Handling of "other" series is moving out of `TimeSeriesWidgetVisualization` for now. I might bring it back later, but this is a ... leaky thing to try and look at.